### DISCUSSION TITLE
Automatic Amoled Theme

### DISCUSSION THREAD

**Discussion by Andrewblasco** [tuskyapp/Tusky#2967] [General]
First of all, thank you so much for this application.

My issue is as follows. I have the app theme set "as system". But when night comes (full of horrors), the theme changes to dark grey. I would prefer dark black for my amoled screen. But is not posible now.

Could you add a simple setting to select which kind of night theme would the user prefer, grey or black?

**Comment by connyduck** [comment]
Yes, we should do that. There is already an issue for that https://github.com/tuskyapp/Tusky/issues/1222
